### ISSUE TITLE
Servo detach does not free timer

### ISSUE THREAD

**Issue by Octogonapus** [CommonWealthRobotics/bowler-rpc-arduino#1]
For some reason calling `Servo.detach()` does not release the timer that servo was using. Adding more servos causes more timers to be claimed until the hardware is exhausted and the servos fail.

**Comment by madhephaestus** [comment]
See the issue in ESP32Servo for resolution

**Comment by Octogonapus** [comment]
Is this fixed in https://github.com/madhephaestus/ESP32Servo/releases/tag/0.4.2? Then you should close https://github.com/madhephaestus/ESP32Servo/issues/5 and publish `0.4.2`.

**Comment by madhephaestus** [comment]
this is fixed in 0.5.0

**Comment by madhephaestus** [comment]
I re-tooled the pubic API in rev 0.6.0, removed public access from a bunch of functions.
